### PR TITLE
#153023596 Add reset password endpoint

### PR DESCRIPTION
**Pull request by betsybeth**

### CODE REVIEW

**Comment by meshnesh** [comment]
* good implementation of the programming logic
* great code and test coverage

**Comment by frankip** [comment]
thumbs up on the coverage and the implementation

File: app/views.py [hunk 1]
```diff
@@ -11,42 +11,49 @@
...
     """Takes a request and returns a json response"""
-    email = request.args.get("email")
-    password = request.args.get("password")
+    json_dict = request.get_json()
+    email = json_dict["email"]
+    password = json_dict["password"]
     for user in users:
-        if email and user.email != email:
+        if user.email != email:
+            print(">>>>>", user.email, email)
```
Comment (muchai-mercy) [inline app/views.py:56]: Remove code used to debug

File: tests/tests_flask.py [hunk 2]
```diff
@@ -1,70 +1,71 @@
...
-                   "author": "beth","location": "nairobi"}
-        resp = self.app.put('/api/v1/events/123456', data=user_data)
-        self.assertEqual(resp.status_code, 200)
-        self.assertIn("event successfully edited", str(resp.data))
-    def test_delete_events(self):
-        self.test_login()
-        resp = self.app.delete('/api/v1/events/123456', data=self.event)
-        self.assertEqual(resp.status_code, 200)
-        self.assertIn("event deleted", str(resp.data))
+# import json
```
Comment (muchai-mercy) [inline tests/tests_flask.py:1]: Remove commented out code

**Pull request merged**